src/tracks/SampleTrack.cpp

```diff
@@ -593,7 +603,24 @@ void SampleTCOView::paintEvent( QPaintEvent * pe )
 		p.fillRect( rect(), c );
 	}
 
-	p.setPen( !muted ? painter.pen().brush().color() : mutedColor() );
+	auto tcoColor = m_tco->hasColor()
+				? m_tco->usesCustomClipColor()
+					? m_tco->color()
+					: m_tco->getTrack()->color()
+				: painter.pen().brush().color();
```

Comment:
I think this might be more readable while using less tabs. I'm aware the extra parenthesis are unnecessary, but to my eyes it groups it better.
```suggestion
	auto tcoColor =
		m_tco->hasColor()
		? (m_tco->usesCustomClipColor()
			? m_tco->color()
			: m_tco->getTrack()->color())
		: painter.pen().brush().color();
```

Reply:
Alternatively
```suggestion
	auto clipColor = m_tco->usesCustomClipColor() ? m_tco->color() : m_tco->getTrack()->color();
	auto tcoColor = m_tco->hasColor() ? clipColor : painter.pen().brush().color();
```